cpu/cpu_darwin_cgo.go

```diff
@@ -8,7 +8,10 @@ package cpu
 #include <sys/sysctl.h>
 #include <sys/mount.h>
 #include <mach/mach_init.h>
+#ifndef TEST_H_
```

Comment:
this is a weird hack to make our Datadog/gopsutil and the shirou/gopsutil not conflict on duplicate symbols. Otherwise you get an error like:

```
duplicate symbol __cgo_8984c9e68292_Cfunc_host_statistics in:
    /var/folders/7f/_5m_jnb57651wk6835y2f53c0000gn/T/go-link-243183959/000005.o
    /var/folders/7f/_5m_jnb57651wk6835y2f53c0000gn/T/go-link-243183959/000009.o
duplicate symbol __cgo_8984c9e68292_Cfunc_mach_host_self in:
    /var/folders/7f/_5m_jnb57651wk6835y2f53c0000gn/T/go-link-243183959/000005.o
    /var/folders/7f/_5m_jnb57651wk6835y2f53c0000gn/T/go-link-243183959/000009.o
ld: 6 duplicate symbols for architecture x86_64
```